crypto/objects/objects.txt

```diff
@@ -933,11 +933,39 @@ nist_hashalgs 1		: SHA256		: sha256
 nist_hashalgs 2		: SHA384		: sha384
 nist_hashalgs 3		: SHA512		: sha512
 nist_hashalgs 4		: SHA224		: sha224
+nist_hashalgs 5		: id-sha512-224		: sha512-224
+nist_hashalgs 6		: id-sha512-256		: sha512-256
```

Comment:
I was commenting at the time just on the specific ones above - however the comment also applies to all the additions where you are adding an id- where there isn't an id- for the existing equivalents. I see no reason to add an "id-" on the front where we don't have and id- for the SHA1 and SHA2 versions.